.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+name: Release Workflow
+
+on:
+  push:
+    tags: [ '**' ]
+  workflow_dispatch:
+
+concurrency: ci-${{ github.ref }}
+
+jobs:
+
+  build-and-run-tests:
+    name: Build and Run Tests
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Set Branch Name
+        uses: nelonoel/branch-name@v1.0.1
+
+      - name: Set GitHub Tag environment variable
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
+        run: |
+          TAG_ID=${GITHUB_REF##*/}
+          echo "ACTIONS_TAG=$TAG_ID" >> $GITHUB_ENV
+
+      - name: Install Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt-hotspot'
+          java-version: '8'
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Copyright Check
+        run: ./copyrightCheck.sh
+
+      - name: Gradle Build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_ACTOR: ${{ secrets.GITHUB_ACTOR }}
+        run: |
+          ./gradlew clean build shadowJar
```

Comment:
I think you can delete this step and just publish.

Reply:
> I think you can delete this step and just publish.

Yeah, I think that makes sense, too. Because we don't need a shadowJar here and the `publish` target already includes doing calling the `build` target.